build-p4ovs.sh

```diff
@@ -41,9 +41,9 @@ fi
 ./boot.sh || exit 1
 if [ ! -z "$DEPS_INSTALL_PATH" ]
 then
-    ./configure --prefix=$DEPS_INSTALL_PATH --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0 -g' --disable-ssl --with-sai || exit 1
+    ./configure --prefix=$DEPS_INSTALL_PATH --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0' --disable-ssl --with-sai || exit 1
 else
-    ./configure --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0 -g' --with-sai || exit 1
+    ./configure --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0' --with-sai || exit 1
```

Comment:
The commit message says disable-ssl for default path too, I think that option is missed on line 46

Reply:
You are right., missed adding disable-ssl in else case. Will add in next patch